build-system/babel-plugins/babel-plugin-transform-fix-leading-comments/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = function() {
       Statement(path) {
         const {node} = path;
         const {trailingComments} = node;
-        if (!trailingComments) {
+        if (!trailingComments || trailingComments.length <= 0) {
```

Comment:
This transform was executing on empty array lengths, addressed since this PR also includes a new transform doing something very similar.

Reply:
just for my own knowledge, can the comments property actually be `null`?